src/auto_archiver/modules/tiktok_tikwm_extractor/tiktok_tikwm_extractor.py

```diff
@@ -0,0 +1,73 @@
+import requests
+from loguru import logger
+from datetime import datetime, timezone
+
+from auto_archiver.core import Extractor
+from auto_archiver.core import Metadata, Media
+
+
+class TiktokTikwmExtractor(Extractor):
+    """
+    Extractor for TikTok that uses an unofficial API and can capture content that requires a login, like sensitive content.
+    """
+
+    def download(self, item: Metadata) -> Metadata:
+        url = item.get_url()
+
+        # detect URLs that we definitely cannot handle
+        if 'tiktok.com' not in item.netloc:
+            return False
+
+        endpoint = f"https://www.tikwm.com/api/?url={url}"
```

Comment:
Might be nice to declare this in the class with TIKWM_ENDPOINT for readability/maintainability (if they change the API URL)


```python

class TiktokTikwmExtractor(Extractor):
    """
    Extractor for TikTok that uses an unofficial API and can capture content that requires a login, like sensitive content.
    """

   TIKWM_ENDPOINT = 'https://www.tikwm.com/api/?url={url}'

   ...
   endpoint = self.TIKWM_ENDPOINT.format(url=url)
   ...
```